client/my-sites/plan-features/header.jsx

```diff
@@ -170,7 +170,7 @@ export class PlanFeaturesHeader extends Component {
 		if ( ! planMatches( planType, { group: GROUP_WPCOM, term: TERM_ANNUALLY } ) ) {
 			return null;
 		}
-		if ( ! isFreePlan( currentSitePlan.productSlug ) ) {
+		if ( ! currentSitePlan || ! isFreePlan( currentSitePlan.productSlug ) ) {
```

Comment:
Not a big deal, but just noting that this could also be written as:

```suggestion
		if ( ! isFreePlan( currentSitePlan?.productSlug ) ) {
```

Reply:
Optional chaining if you are interested what this is @dzver :)